src/applications/coronavirus-research/config/form.js

```diff
@@ -1,29 +1,31 @@
 import environment from 'platform/utilities/environment';
-import fullSchema from 'vets-json-schema/dist/COVID-VACCINE-TRIAL-schema.json';
+// import fullSchema from 'vets-json-schema/dist/COVID-VACCINE-TRIAL-schema.json';
+import fullSchema from './temp.json';
 
 import definitions from 'vets-json-schema/dist/definitions.json';
 
 import IntroductionPage from '../containers/IntroductionPage';
 import ConfirmationPage from '../containers/ConfirmationPage';
 
 import { uiSchema } from '../pages/covidResearchUISchema';
+import { transformForSubmit } from 'platform/forms-system/src/js/helpers';
 
 const { fullName, email, usaPhone, date, usaPostalCode } = definitions;
 
 const checkBoxElements = [
-  'healthHistory',
-  'transportation',
-  'employmentStatus',
-  'gender',
-  'raceEthnicityOrigin',
+  'HEALTH_HISTORY',
```

Comment:
The change in casing here is to indicate the special treatment of the element?

Reply:
Yes it aligns with how the elements are coded in the schema to provide a `namespace` for the `Select Group` elements.  I wanted to be consistent with the Schema naming so it is like `SELECT_NAME_SPACE`::`OPTION_ONE`. This way it is easy to parse and know which `Select Group` the user just modified, and take appropriate action to handle the `NONE_OF_THE_ABOVE` option.